### PR TITLE
Update README.md with documentation

### PR DESCRIPTION
**Pull request by marijnschilling**
I haven't wrote documentation before, so feedback is very much welcome :v:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +14,53 @@
 
 ## Usage
 
+#### DateScrubber (class)
+```swift
+    // initialise the dateScrubber
+    let dateScrubber = DateScrubber()
+    
+    //set the collectionViewController as the delegate of the dateScrubber
+    public var delegate : DateScrubberDelegate?
```
Comment (elland) [inline README.md]: Shouldn't this be `weak`? 💭

**Comment by elland** [comment]
I've some ideas, but not enough time atm. 😦

**Comment by 3lvis** [comment]
The gif and the introduction are sweet. The `Usage` might need to be taken in a different approach though. I'm thinking more on _"what steps should I take in order to get this working?"_.

This is not final but something like this:

https://github.com/3lvis/Viewer/blob/master/README.md

**Comment by elland** [comment]
@3lvis also grammar. 😢

**Comment by marijnschilling** [comment]
I'll work on it! :v: Thanks!

**Comment by 3lvis** [comment]
This works, go for it! 🚀

**Pull request merged**